src/index.html

```diff
@@ -7,6 +7,422 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="./images/logo.png" alt="MOYO Logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+        >
+          Laptops&nbsp;&&nbsp;Computers
```

Comment:
Fix the code style, please 
```suggestion
              data-qa="nav-hover"
            >
              Laptops&nbsp;&&nbsp;Computers
```